# YX-AUG Motif Ancestry Data Files

## __Initial Files__
(Uploaded Feb 26 2020 to Box, later uploaded to Slack on June 25 2020 by Helena)

- __picornaviridae_aligned.fasta__: 226 aligned nucleotide sequences of the complete genome sequence of members of the Picornaviridae family. No outgroup present.
- __potyviridae_aligned.fasta__: 218 aligned nucleotide sequences of the complete genome sequence of members of the Potyviridae family. No outgroup present.
- __picornaviridae_5UTR_aligned.fas__: 129 aligned nucleotide sequences of the 5'UTR sequence of members of the Picornaviridae family. No outgroup present.
- __potyviridae_5UTR_aligned.fas__: 152 aligned nucleotide sequences of the 5'UTR sequence of members of the Potyviridae family. No outgroup present.

## __Complete Genome Files__
(Uploaded April 24 2020 to Slack by Helena)  

- __picornaviridae_genome_with_outgroup_aligned.fasta__: 228 aligned nucleotide sequences of the complete genome sequence of members of the Picornaviridae family. 
Vesicular exanthema of swine virus and Norovirus (Caliciviridae family) are added as outgroups. 
- __potyviridae_genome_with_outgroup_aligned.fasta__: 220 aligned nucleotide sequences of the complete genome sequence of members of the Potyviridae family. 
Potato leafroll virus and Barley yellow dwarf virus (Luteoviridae family) are added as outgroups. 
- __picornaviridae_potyviridae_genome_aligned.fasta__: 446 aligned nucleotide sequences of the complete genome of members of the Potyviridae family and the Picornaviridae family. Vesicular exanthema of swine virus and Norovirus (Caliciviridae family) are added as outgroups.

## __5' UTR Files__
(Uploaded April 24 2020 to Slack by Helena)  
- __picornaviridae_5UTR_with_outgroup_aligned.fasta__: 131 aligned nucleotide sequences of the 5' UTR sequence of members of the Picornaviridae family. 
Vesicular exanthema of swine virus and Norovirus (Caliciviridae family) are added as outgroups. 
- __potyviridae_5UTR_with_outgroup_aligned.fasta__: 154 aligned nucleotide sequences of the 5' UTR sequence of members of the Potyviridae family. 
Potato leafroll virus and Barley yellow dwarf virus (Luteoviridae family) are added as outgroups. 
- __potyviridae_picornaviridae_5UTR.fasta__: 283 aligned nucleotide sequences of the 5' UTR sequence of members of the Potyviridae family and the Picornaviridae family. Vesicular exanthema of swine virus and Norovirus (Caliciviridae family) are added as outgroups.

## __Replicase Files__
(Uploaded May 18 2020 to Slack by Helena)  

- __replicase_potyvirus_nucleotide_aligned.fasta__: 120 aligned nucleotide sequences of the replicase gene of members of the Potyviridae family. Potato leafroll virus and Barley yellow dwarf virus (Luteoviridae family) are added as outgroups. 
- __replicase_potyvirus_protein_aligned.fasta__: 120 aligned amino acid sequences of the replicase protein of members of the Potyviridae family. Potato leafroll virus and Barley yellow dwarf virus (Luteoviridae family) are added as outgroups. 
- __replicase_picornavirus_nucleotide_aligned.fasta__: 86 aligned nucleotide sequences of the replicase gene of members of the Picornaviridae family. Potato leafroll virus and Barley yellow dwarf virus (Luteoviridae family) are added as outgroups. 
- __replicase_picornavirus_protein_aligned.fasta__: 86 aligned amino acid sequences of the replicase protein of members of the Picornaviridae family. Potato leafroll virus and Barley yellow dwarf virus (Luteoviridae family) are added as outgroups. 
- __replicase_picornavirus_and_potyvirus_nucleotide_aligned.fasta__: 204 aligned nucleotide sequences of the replicase gene of members of the Potyviridae and Picornaviridae families. Potato leafroll virus and Barley yellow dwarf virus (Luteoviridae family) are added as outgroups. 
- __replicase_picornavirus_and_potyvirus_protein_aligned.fasta__: 204 aligned amino acid sequences of the replicase protein of members of the Potyviridae and Picornaviridae families. Potato leafroll virus and Barley yellow dwarf virus (Luteoviridae family) are added as outgroups. 

## __Replicase Files (with new Mitovirus outgroup)__
(Uploaded Aug 5 2020 to Slack by Helena)  

- __replicase_potyvirus_nucleotide_mitovirus_outgroup_aligned.fasta__: 120 aligned nucleotide sequences of the replicase gene of members of the Potyviridae family. Cryphonectria parasitica mitovirus and Ophiostoma mitovirus 4 (Narnaviridae family) are added as outgroups.  
- __replicase_potyvirus_protein_mitovirus_outgroup_aligned.fasta__: 120 aligned amino acid sequences of the replicase gene of members of the Potyviridae family. Cryphonectria parasitica mitovirus and Ophiostoma mitovirus 4 (Narnaviridae family) are added as outgroups. 
- __replicase_picornavirus_nucleotide_mitovirus_outgroup_aligned.fasta__: 86 aligned nucleotide sequences of the replicase gene of members of the Picornaviridae family. Cryphonectria parasitica mitovirus and Ophiostoma mitovirus 4 (Narnaviridae family) are added as outgroups.
- __replicase_picornavirus_protein_mitovirus_outgroup_aligned.fasta__: 86 aligned amino acid sequences of the replicase gene of members of the Picornaviridae family. Cryphonectria parasitica mitovirus and Ophiostoma mitovirus 4 (Narnaviridae family) are added as outgroups.
- __replicase_picornavirus_and_potyvirus_nucleotide_mitovirus_outgroup_aligned.fasta__: 204 aligned nucleotide sequences of the replicase gene of members of the Potyviridae and Picornaviridae family. Cryphonectria parasitica mitovirus and Ophiostoma mitovirus 4 (Narnaviridae family) are added as outgroups.  
- __replicase_picornavirus_and_potyvirus_protein_mitovirus_outgroup_aligned.fasta__: 204 aligned amino acid sequences of the replicase gene of members of the Potyviridae and Picornaviridae family. Cryphonectria parasitica mitovirus and Ophiostoma mitovirus 4 (Narnaviridae family) are added as outgroups.  

## __5' UTR Files with Mitovirus outgroup__
(Uploaded Aug 12 2020 to Slack by Helena)  
- __picornaviridae_5UTR_mitovirus_outgroup_aligned.fasta__: 131 aligned nucleotide sequences of the 5' UTR sequence of members of the Picornaviridae family. 
Cryphonectria parasitica mitovirus and Ophiostoma mitovirus 4 (Narnaviridae family) are added as outgroups.
- __potyviridae_5UTR_mitovirus_outgroup_aligned.fasta__: 154 aligned nucleotide sequences of the 5' UTR sequence of members of the Potyviridae family. 
Cryphonectria parasitica mitovirus and Ophiostoma mitovirus 4 (Narnaviridae family) are added as outgroups.
- __potyviridae_picornaviridae_5UTR_mitovirus_outgroup_aligned.fasta__: 283 aligned nucleotide sequences of the 5' UTR sequence of members of the Potyviridae family and the Picornaviridae family. Cryphonectria parasitica mitovirus and Ophiostoma mitovirus 4 (Narnaviridae family) are added as outgroups.

## __Replicase & 5'UTR Picornaviridae plus Triticum mosaic virus Files (with Mitovirus outgroup)__
(Uploaded Aug 27 2020 to Slack by Helena)  

- __replicase_picornavirus_triticum_nucleotide_mitovirus_outgroup_aligned.fasta__: 86 aligned amino acid sequences of the replicase gene of members of the Picornaviridae family plus the potyvirus Triticum mosaic virus (87 sequence in total). Cryphonectria parasitica mitovirus and Ophiostoma mitovirus 4 (Narnaviridae family) are added as outgroups. 
- __replicase_picornavirus_triticum_protein_mitovirus_outgroup_aligned.fasta__: 86 aligned amino acid sequences of the replicase gene of members of the Picornaviridae family plus the potyvirus Triticum mosaic virus (87 sequence in total). Cryphonectria parasitica mitovirus and Ophiostoma mitovirus 4 (Narnaviridae family) are added as outgroups.
- __picornaviridae_triticum_5UTR_mitovirus_outgroup_aligned.fasta__: 131 aligned nucleotide sequences of the 5' UTR sequence of members of the Picornaviridae family plus the potyvirus Triticum mosaic virus (132 sequences total). Cryphonectria parasitica mitovirus and Ophiostoma mitovirus 4 (Narnaviridae family) are added as outgroups.

## __Complete Genome Files with Mitovirus Outgroup__
(Uploaded Sep 8 2020 to Slack by Helena)  

- __picornaviridae_genome_mitovirus_outgroup_aligned.fasta.fasta__: 228 aligned nucleotide sequences of the complete genome sequence of members of the Picornaviridae family. Cryphonectria parasitica mitovirus and Ophiostoma mitovirus 4 (Narnaviridae family) are added as outgroups.
Vesicular exanthema of swine virus and Norovirus (Caliciviridae family) are added as outgroups. 
- __potyviridae_genome_mitovirus_outgroup_aligned.fasta__: 220 aligned nucleotide sequences of the complete genome sequence of members of the Potyviridae family. 
Cryphonectria parasitica mitovirus and Ophiostoma mitovirus 4 (Narnaviridae family) are added as outgroups.
- __picornaviridae_potyviridae_genome_mitovirus_outgroup_aligned.fasta__: 446 aligned nucleotide sequences of the complete genome of members of the Potyviridae family and the Picornaviridae family. Cryphonectria parasitica mitovirus and Ophiostoma mitovirus 4 (Narnaviridae family) are added as outgroups.

## __Files 2021 (new approach, smaller datasets)__

Uploaded March 24 2021 to Slack by Helena 
- __flaviviridae_w_outgroup_wo_flavivirus_aligned.fasta__: 8 aligned nucleotide sequences of the 5' UTR of members of the Flaviviridae family with a characterized IRES plus 1 outgroup (9 sequence total: 4 Hepacivirus, 2 Pestivirus, 2 Pegivirus, 1 outgroup). Ophiostoma mitovirus 4 (Narnaviridae family) is added as outgroup.

- __potyviridae_poacevirus_w_outgroup_aligned.fasta__: 3 aligned nucleotide sequences of the 5' UTR of members of the genus Poacevirus (Potyviridae family) including TriMV which has a characterized IRES (4 sequence total: 3 Poacevirus, 1 outgroup). Ophiostoma mitovirus 4 (Narnaviridae family) is added as outgroup.

Uploaded March 30 2021 to Slack by Helena
- __picornaviridae_w_outgroup_aligned.fasta__: 10 aligned nucleotide sequences of the 5' UTR of members of Picornaviridae family with a characterized IRES plus 1 outgroup (11 sequence total: 5 Enterovirus, 2 Aphotvirus, 1 Hepatovirus, 2 Cardiovirus, 1 outgroup). Ophiostoma mitovirus 4 (Narnaviridae family) is added as outgroup.

Uploaded May 10 2021 to Slack by Helena
- __dicistrovirus_I_w_outgroup_aligned.fasta__: 7 aligned nucleotide sequences of the 5' UTR of members of Dicistroviridae family with a characterized IRES plus 1 outgroup (8 sequence total: 4 Cripavirus, 3 Aparavirus, 1 outgroup). Ophiostoma mitovirus 4 (Narnaviridae family) is added as outgroup.
- __dicistrovirus_II_w_outgroup_aligned.fasta__: 7 aligned nucleotide sequences of the intergenic region of members of Dicistroviridae family with a characterized IRES plus 1 outgroup (8 sequence total: 4 Cripavirus, 3 Aparavirus, 1 outgroup). Ophiostoma mitovirus 4 (Narnaviridae family) is added as outgroup.

Uploaded June 21 2021 to Slack by Helena
- __Picornaviridae_w_triticum_w_outgroup_aligned.fasta__: 10 aligned nucleotide sequences of the 5' UTR of members of Picornaviridae family with a characterized IRES, plus Triticum mosaic virus (TriMV), plus 1 outgroup (12 sequence total: 5 Enterovirus, 2 Aphotvirus, 1 Hepatovirus, 2 Cardiovirus, 1 TriMV, 1 outgroup). Ophiostoma mitovirus 4 (Narnaviridae family) is added as outgroup.

- __Dicistrovirus_I_w_triticum_w_outgroup_aligned.fasta__: 7 aligned nucleotide sequences of the 5' UTR of members of Dicistroviridae family with a characterized IRES, plus Triticum mosaic virus (TriMV), plus 1 outgroup (9 sequence total: 4 Cripavirus, 3 Aparavirus, 1 TriMV, 1 outgroup). Ophiostoma mitovirus 4 (Narnaviridae family) is added as outgroup.

- __Dicistrovirus_II_w_triticum_w_outgroup_aligned.fasta__: 7 aligned nucleotide sequences of the intergenic region of members of Dicistroviridae family with a characterized IRES, plus Triticum mosaic virus (TriMV), plus 1 outgroup (9 sequence total: 4 Cripavirus, 3 Aparavirus, 1 TriMV, 1 outgroup). Ophiostoma mitovirus 4 (Narnaviridae family) is added as outgroup.


# Script/results files
## Initial four files 
Analysis of 4 files in `scripts/virus_project_initial_files.Rmd`
with output files in `initial-files` folder in google drive:
Tree files:
- RAxML_bestTree_picornaviridae_5UTR_aligned_midpoint.tree
- RAxML_bestTree_picornaviridae_5UTR_aligned.tree
- RAxML_bestTree_picornaviridae_aligned_midpoint.tree
- RAxML_bestTree_picornaviridae_aligned.tree
- RAxML_bestTree_potyviridae_5UTR_aligned_midpoint.tree
- RAxML_bestTree_potyviridae_5UTR_aligned.tree
- RAxML_bestTree_potyviridae_aligned_midpoint.tree
- RAxML_bestTree_potyviridae_aligned.tree

Ancestral Sequence files:

- picornaviridae_5UTR_aligned_midpoint_AncestralSeq.fasta
- potyviridae_5UTR_aligned_midpoint_AncestralSeq.fasta

These analysis do not have an outgroup and we use midpoint rooting to root the estimated trees.

## Complete genomes and 5UTR files
Analysis in `scripts/virus_project_genome_5UTR.md`
with output files in `5UTR_outgroup` folder in google drive:

Tree files:
- RAxML_bestTree_picornaviridae_5UTR_with_outgroup_aligned_midpoint.tree
- RAxML_bestTree_picornaviridae_5UTR_with_outgroup_aligned_unrooted.tree
- RAxML_bestTree_picornaviridae_potyviridae_5UTR_aligned_midpoint.tree
- RAxML_bestTree_picornaviridae_potyviridae_5UTR_aligned_unrooted.tree
- RAxML_bestTree_potyviridae_5UTR_with_outgroup_aligned_midpoint.tree
- RAxML_bestTree_potyviridae_5UTR_with_outgroup_aligned_unrooted.tree

Ancestral Sequence files:

- picornaviridae_5UTR_with_outgroup_aligned_midpoint_AncestralSeq.fasta
- picornaviridae_potyviridae_5UTR_aligned_midpoint_AncestralSeq.fasta
- potyviridae_5UTR_with_outgroup_aligned_midpoint__AncestralSeq.fasta


With results in `complete_genome_outgroup` in google drive folder:

Tree files:
- RAxML_bestTree_picornaviridae_genome_with_outgroup_aligned_midpoint.tree
- RAxML_bestTree_picornaviridae_genome_with_outgroup_aligned_unrooted.tree
- RAxML_bestTree_picornaviridae_potyviridae_genome_aligned_midpoint.tree
- RAxML_bestTree_picornaviridae_potyviridae_genome_aligned_unrooted.tree
- RAxML_bestTree_potyviridae_genome_with_outgroup_aligned_midpoint.tree
- RAxML_bestTree_potyviridae_genome_with_outgroup_aligned_unrooted.tree

Ancestral Sequence files:

- picornaviridae_genome_with_outgroup_aligned_midpoint_AncestralSeq.fasta
- picornaviridae_potyviridae_genome_aligned_midpoint_AncestralSeq.fasta
- potyviridae_genome_with_outgroup_aligned_midpoint_AncestralSeq.fasta

## First replicase files

Analysis of 6 files in `scripts/replicase_outgroup_part1.md` and `scripts/replicase_outgroup_part3.md`
with output files in `replicase_two_outgroup` folder in google drive. This folder contains the following subfolder:

1. With the `tree_deletion_extra_taxa` taxa: it contains the tree with the deleted taxa not in replicase files but in 5UTR files

-"picornavirus_nucleotide_midpoint_replicasetree_deletion.tree"
-"picornavirus_nucleotide_outgroup_replicasetree_deletion.tree"
-"picornavirus_potyvirus_nucleotide_midpoint_replicasetree_deletion.tree"
-"potyvirus_nucleotide_midpoint_replicasetree_deletion.tree"
-"potyvirus_nucleotide_outgroup_replicasetree_deletion.tree"

2. With the `sequence_match_replicase_5UTR` : it contains the 5UTR sequence which ID matches the replicase files


3. With `Ancestral_Sequence_5UTR_replicasetree`

it shows the ancestral reconstruction sequence of the nucleotide sequence based on 5UTR files and replicase tree

-"picornaviridae_5UTR_replicasetree_midpoint_AncestralSeq.fasta"
-"picornaviridae_5UTR_replicasetree_outgroup_AncestralSeq.fasta"
-"picornaviridae_potyviridae_5UTR_replicasetree_midpoint_AncestralSeq.fasta"
-"potyviridae_5UTR_replicasetree_midpoint_AncestralSeq.fasta"
-"potyviridae_5UTR_replicasetree_outgroup_AncestralSeq"



## Second replicase files (mitovirus outgroup)

Analysis of 6 files in `scripts/replicase_mitovirus_outgroup.md`
with output files in `replicase_mitovirus` folder in google drive. This folder contains the following subfolder:

1. With `original_sequence_alignment_replicase` and `original_sequence_alignment_replicase_5UTR` folder: This folder contains the original sequence alignment of six replicase files and 5UTR files.

2. With `revised_sequence_alignment_replicase` folder: This folder contains the sequence alignment of six replicase files, the difference is that the space and special characters in the ID of virus are removed. 

- replicase_picorna_nucleotide_mito_nospace.fasta
- replicase_picorna_protein_mito_nospace.fasta
- replicase_picornapoty_nucleotide_mito_nospace.fasta
- replicase_picornapoty_protein_mito_nospace.fasta
- replicase_poty_nucleotide_mito_nospace.fasta
- replicase_poty_protein_mito_nospace.fasta

and the replicase contains only the taxa that exist in 5UTR

- replicase_picornavirus_5UTR_mito_comb.fasta
- replicase_picornapotyvirus_5UTR_mito_comb.fasta
- replicase_potyvirus_5UTR_mito_comb.fasta

3. With `revised_sequence_alignment_5UTR folder`: this folder contains the sequence alignment of the 5UTR which delete the taxa not in the replicase file and same ID as the replicase file

- picornaviridae_5UTR_mitovirus_outgroup_ReplicaseID.fasta
- potyviridae_5UTR_mitovirus_outgroup_ReplicaseID.fasta
- potyviridae_picornaviridae_5UTR_mitovirus_outgroup_ReplicaseID.fasta

the sequence alignment of the 5UTR and the space and special characters in the ID of virus are removed

- picornaviridae_5UTR_mitovirus_outgroup_ReplicaseID_nospace.fasta
- potyviridae_5UTR_mitovirus_outgroup_ReplicaseID_nospace.fasta
- potyviridae_picornaviridae_5UTR_mitovirus_outgroup_ReplicaseID_nospace.fasta

With the deletion on gap-only sites

- picornaviridae_5UTR_mitovirus_outgroup_ReplicaseID_nospacegap.fas
- potyviridae_5UTR_mitovirus_outgroup_ReplicaseID_nospacegap.fas
- potyviridae_picornaviridae_5UTR_mitovirus_outgroup_ReplicaseID_nospacegap.fas

4. Within `Trees` folder, there are other subfolders:

`tree_raxml_output` folder: it contains the tree files derived from RAXML:

- replicase_picornavirus_and_potyvirus_nucleotide_mito.tree
- replicase_picornavirus_and_potyvirus_protein_mito.tree
- replicase_picornavirus_nucleotide_mito.tree
- replicase_picornavirus_protein_mito.tree
- replicase_potyvirus_nucleotide_mito.tree
- replicase_potyvirus_protein_mito.tree

`tree_raxml_rooting` folder: it contains the trees after rooting at midpoint or outgroup

- replicase_picornavirus_and_potyvirus_nucleotide_mito_midpoint.tree
- replicase_picornavirus_and_potyvirus_nucleotide_mito_outgroup.tree
- replicase_picornavirus_and_potyvirus_protein_mito_midpoint.tree
- replicase_picornavirus_and_potyvirus_protein_mito_outgroup.tree
- replicase_picornavirus_nucleotide_mito_midpoint.tree
- replicase_picornavirus_protein_mito_midpoint.tree
- replicase_picornavirus_protein_mito_outgroup.tree
- replicase_potyvirus_nucleotide_mito_midpoint.tree
- replicase_potyvirus_nucleotide_mito_outgroup.tree
- replicase_potyvirus_protein_mito_midpoint.tree
- replicase_potyvirus_protein_mito_outgroup.tree

`tree_deleted_taxa` folder: it contains tress that delete the taxa not in the 5UTR

- picornavirus_and_potyvirus_nucleotide_mitovirus_replicasetree_midpoint_deletion.tree
- picornavirus_and_potyvirus_nucleotide_mitovirus_replicasetree_outgroup_deletion.tree
- picornavirus_nucleotide_mitovirus_replicasetree_midpoint_deletion.tree
- picornavirus_nucleotide_mitovirus_replicasetree_outgroup_deletion.tree
- potyvirus_nucleotide_mitovirus_replicasetree_midpoint_deletion.tree
- potyvirus_nucleotide_mitovirus_replicasetree_outgroup_deletion.tree


5. With `Ancestral_Sequence_mitovirus` folder: it contains the ancestral sequences

- picornaviridae_5UTR_mitovirus_replicasetree_midpoint_AncestralSeq.fasta
- picornaviridae_5UTR_mitovirus_replicasetree_outgrouprooting_AncestralSeq.fasta
- picornaviridae_and_potyviridae_5UTR_mitovirus_replicasetree_midpoint_AncestralSeq.fasta
- picornaviridae_and_potyviridae_5UTR_mitovirus_replicasetree_outgroup_AncestralSeq.fasta
- potyviridae_5UTR_mitovirus_replicasetree_midpoint_AncestralSeq.fasta
- potyviridae_5UTR_mitovirus_replicasetree_outgroup_AncestralSeq.fasta
- potyviridae_mitovirus_protein_midpoint_AncestralSeq.fasta
- potyviridae_mitovirus_protein_outgrouproot_AncestralSeq.fasta
- picornaviridae_mitovirus_protein_midpoint_AncestralSeq.fasta
- picornaviridae_mitovirus_protein_outgrouproot_AncestralSeq.fasta

## __Replicase & 5'UTR Picornaviridae plus Triticum mosaic virus Files (with Mitovirus outgroup)

With `Tree` folder: it contains the phylogenetic tree files directly derived from RAXML and midpoint and outgroup rooting on the trees

- replicase_picornavirus_triticum_nucleotide_mitovirus_midpoint.tree
- replicase_picornavirus_triticum_nucleotide_mitovirus_outgroup_noroot.tree
- replicase_picornavirus_triticum_nucleotide_mitovirus_outgrouprooting.tree
- replicase_picornavirus_triticum_protein_mitovirus_midpoint.tree
- replicase_picornavirus_triticum_protein_mitovirus_outgroup_noroot.tree
- replicase_picornavirus_triticum_protein_mitovirus_outgrouprooting.tree

Within `Tree` folder, there is a subfolder called `tree_deletion`: it contains the tree of the nucleotide files which delelet the taxa that are not in 5TUR

- picornavirus_triticum_nucleotide_mitovirus_replicasetree_midpoint_deletion.tree
- picornavirus_triticum_nucleotide_mitovirus_replicasetree_outgroup_deletion.tree

Within `sequence_revised`: it contains the sequences whose format is revised. The main revision is the ID of each virus. For instance, the space in ID, gap-only columns, and the matched ID between 5UTR files and replicase files

- picornaviridae_triticum_5UTR_mitovirus_outgroup_ReplicaseID_nospace.fasta
- picornaviridae_triticum_5UTR_mitovirus_outgroup_ReplicaseID_nospacegap.fas
- picornaviridae_triticum_5UTR_mitovirus_outgroup_ReplicaseID.fasta
- replicase_picornavirus_triricum_nucleotide_5UTR_mito_combination.fasta
- replicase_picornavirus_triticum_nucleotide_mitovirus_outgroup_nospace.fasta
- replicase_picornavirus_triticum_protein_mitovirus_outgroup_nospace.fasta
- replicase_picornavirus_triticum_protein_mitovirus_outgroup_nospaceX.fas


With`Ancestral Sequences` folder: it contains the ancestral sequences of two rooting ways

- picornaviridae_triticum_5UTR_mitovirus_replicasetree_midpoint_AncestralSeq.fasta
- picornaviridae_triticum_5UTR_mitovirus_replicasetree_outgroup_AncestralSeq.fasta
- replicase_picornavirus_triticum_protein_mitovirus_midpoint_AncestralSeq.fasta
- replicase_picornavirus_triticum_protein_mitovirus_outgrouprooting_AncestralSeq.fast

## __Complete_Genome_Mitovirus__

Within `Tree` folder: it contains the trees of complete genome

- potyviridae_genome_mitovirus_outgroup_outgrouprooting.tree
- potyviridae_genome_mitovirus_outgroup_midpointrooting.tree
- picornaviridae_potyviridae_genome_mitovirus_outgroup_midpointrooting.tree
- picornaviridae_genome_mitovirus_outgroup_midpointrooting.tree

Within `Ancestral Sequences` folder: it contains the ancestral sequences of the complete genome virus

- potyviridae_genome_mitovirus_outgroup_aligned_outgrouprooting_AncestralSeq.fasta
- potyviridae_genome_mitovirus_outgroup_aligned_midpointrooting_AncestralSeq.fasta
- picornaviridae_genome_mitovirus_outgroup_aligned_midpointrooting_AncestralSeq.fasta


## __Analysis_without_Outgroup__

Within `Trees` folder: it contains the trees with the 5UTR files without the outgroups

- picornaviridae_5UTR_mitovirus_replicasetree_outgrouprooting_nooutgroup.tree
- potyviridae_picornaviridae_5UTR_mitovirus_replicasetree_outgrouprooting_nooutgroup.tree
- potyvirus_nucleotide_mitovirus_replicasetree_outgrouprooting_nooutgroup.tree

Within `revised_5UTR_nooutgroup` folder: it contains the 5UTR files with the ID in the replicase files and the removal of outgroup

- picornaviridae_5UTR_mitovirus_ReplicaseID_nooutgroup.fas
- potyviridae_5UTR_mitovirus_outgroup_ReplicaseID_nooutgroup.fas
- potyviridae_picornaviridae_5UTR_mitovirus_ReplicaseID_nooutgroup.fas

Within `Ancestral_Sequence` folder: it contains the ancestral sequences of the 5UTR files without the outgroup

- picornaviridae_5UTR_mitovirus_replicasetree_outgrouprooting_nooutgroup_AncestralSeq.fasta
- potyviridae_5UTR_mitovirus_replicasetree_outgrouprooting_nooutgroup_AncestralSeq.fasta
- potyviridae_picornaviridae_5UTR_mitovirus_replicasetree_outgrouprooting_nooutgroup_AncestralSeq.fasta

Within `Ancestral Confidence Value` folder: it contains the confidence value of ancestral sequences

- picornaviridae_5UTR_mitovirus_replicasetree_outgrouprooting_nooutgroup_Ancestral_Confidence
- potyviridae_5UTR_mitovirus_replicasetree_outgrouprooting_nooutgroup_Ancestral_Confidence
- potyviridae_picornaviridae_5UTR_mitovirus_replicasetree_outgrouprooting_nooutgroup_Ancestral_Confidence

## __Analysis_With_Or_Without_Triticum__

Within `Trees(outgroup and Triticum removed)` folder: it contains the trees with the 5UTR files without the outgroups and without the (Triticum + Outgroup)
picornaviridae_triticum_5UTR_mitovirus_outgroup_ReplicaseID_nooutgroup.fa
- picornavirus_triticum_nucleotide_mitovirus_replicasetree_outgrouprooting_nooutgroup.tree
- picornavirus_triticum_nucleotide_mitovirus_replicasetree_outgrouprooting_noTriticum.tree
- replicase_picornavirus_triticum_protein_mitovirus_outgrouprooting_nooutgroup.tree
- replicase_picornavirus_triticum_protein_mitovirus_outgrouprooting_noTriticum.tree

Within `Aligned_Sequences (Outgroup or Triticum removed)` folder: it contains the files with the ID in the replicase files and the removal of outgroup and Triticum

- picornaviridae_triticum_5UTR_mitovirus_outgroup_ReplicaseID_nooutgroup.fas
- picornaviridae_triticum_5UTR_mitovirus_outgroup_ReplicaseID_noTriticum.fas
- replicase_picornavirus_triticum_protein_mitovirus_outgroup_nooutgroupfile.fas
- replicase_picornavirus_triticum_protein_mitovirus_outgroup_noTriticumfile.fas

Within `Ancestral Sequences/No_Outgroup_No_Triticum ` folder: it contains the estimated nucleotide and protein ancestral sequences without the outgroup and Triticum during the ancestral reconstruction

- picornaviridae_triticum_5UTR_mitovirus_nucleotide_outgrouprooting_noTriticum_noOutgroup_AncestralSeq.fasta
- replicase_picornavirus_triticum_protein_mitovirus_outgrouprooting_noTriticum_noOutgroup_AncestralSeq.fasta

Within `Ancestral Sequences/No_Outgroup_With_Triticum ` folder: it contains the estimated nucleotide and protein ancestral sequences without the outgroup but keeping the Triticum during the ancestral reconstruction

- picornaviridae_triticum_5UTR_mitovirus_nucleotide_outgrouprooting_with_Triticum_nooutgroup_AncestralSeq.fasta
- replicase_picornavirus_triticum_protein_mitovirus_outgrouprooting_with_Triticum_nooutgroup_AncestralSeq.fasta

Within `Ancestral Confidence Values` folder: it contains the confidence value of ancestral sequences

- picornaviridae_triticum_5UTR_mitovirus_nucleotide_outgrouprooting_noTriticum_noOutgroup_Ancestral_Confidence
- picornaviridae_triticum_5UTR_mitovirus_nucleotide_outgrouprooting_With_Triticum_nooutgroup_Ancestral_Confidence


## Complete_Genome_Without_Outgroups

Within `Trees(outgroup Mitovirus removed)` folder: it contains the tree files of the complete genome without the outgroups (Mitovirus)

-  potyviridae_genome_mitovirus_outgrouprooting_nooutgroup.tree
-  replicase_picornavirus_nucleotide_mito_outgrouprooting_ID_nooutgroup.tree
-  replicase_picornavirus_potyvirus_nucleotide_mitovirus_outgrouprooting_ID_nooutgroup.tree

Within `Nucleotide Sequences (outgroup Mitovirus removed)` folder: it contains the nucleotide sequences of the complete genome with the Mitovirus outgroup removed

-  picornaviridae_genome_mitovirus_aligned_nospacegap_deleted_nooutgroup.fas
-  picornaviridae_potyviridae_genome_mitovirus_nospacegap_deleted_nooutgroup.fas
-  potyviridae_genome_mitovirus_aligned_nospacegap_nooutgroup.fas

Within `Ancestral Sequences` folder: it contains the most probable ancestral sequences of the complete genome without the outgroup at each site

-  picornaviridae_genome_mitovirus_aligned_outgrouprooting_nooutgroup_AncestralSeq.fasta
-  picornaviridae_potyviridae_genome_mitovirus_outgrouprooting_nooutgroup_AncestralSeq.fasta
-  potyviridae_genome_mitovirus_aligned_outgrouprooting_nooutgroup_AncestralSeq.fasta

Within `Ancestral Confidence Values` folder: it contains the confidence value of ancestral sequences at each site

- picornaviridae_genome_mitovirus_aligned_outgrouprooting_nooutgroup_Ancestral_ConfValue


## __New_Dataset__

This folder contains the files of the new dataset with smaller size and the new approach. Reproducible script is in the `scripts_nds.md` file.

Within `Trees` folder: it includes the trees of three viruses families 
Note: The file with label in the name represents that the label of every internal node is included in the trees

- __flaviviridae_wo_outgroup_wo_flavivirus_w_root.tree__
- __flaviviridae_wo_outgroup_wo_flavivirus_w_root_label.tree__
- __picornaviridae_wo_outgroup_w_root.tree__
- __picornaviridae_wo_outgroup_aligned_w_root_label.tree__
- __potyviridae_poacevirus_wo_outgroup_w_root.tree__
- __potyviridae_poacevirus_wo_outgroup_w_root_label.tree__

Within `Joint Sequences` folder: it includes the calculated ancestral sequences at every internal nodes

(N0 represents the ancestral sequences at the rooting point)

- __flaviviridae_wo_outgroup_wo_flavivirus_joint_ancestors.fasta__
- __picornaviridae_wo_outgroup_aligned_joint_ancestors.fasta__
- __potyviridae_poacevirus_wo_outgroup_joint_ancestors.fasta__

Within `Ancestral Confidence Value` folder: it contains the confidence value of the estimated ancestral sequences at every internal nodes for the three datasets

- __flaviviridae_wo_outgroup_wo_flavivirus_joint_ancestors_ConfValue__
- __picornaviridae_wo_outgroup_aligned_joint_ancestors_ConfValue__
- __potyviridae_poacevirus_wo_outgroup_joint_ancestors_ConfValue__

## __dicistrovirus_dataset__

This folder contains the files of the analysis wit the discistrovirus (two datasets total)

Within `trees` folder: it includes the estimated trees of the dicistrovirus I and II viruses (the outgroup is removed)

Note: The file with label in the name represents that the label of every internal node is included in the trees

- __dicistrovirus_I_wo_outgroup_aligned_w_root_w_label.tree__
- __dicistrovirus_I_wo_outgroup_aligned_w_root.tree__
- __dicistrovirus_II_wo_outgroup_aligned_w_root_w_label.tree__
- __dicistrovirus_II_wo_outgroup_w_root.tree__


Within `joint ancestors` folder: it includes the calculated ancestral sequences at every internal nodes

(N0 represents the ancestral sequences at the rooting point)

- __dicistrovirus_I_wo_outgroup_aligned_joint_ancestors.fasta__
- __dicistrovirus_II_wo_outgroup_aligned_joint_ancestors.fasta__

Within `ancestral confidence values` folder: it contains the confidence values of the estimated ancestral sequences at every internal nodes for the two datasets

- __dicistrovirus_I_wo_outgroup_aligned_joint_ancestors_ConfValue__
- __dicistrovirus_II_wo_outgroup_aligned_joint_ancestors_ConfValue__


## __NewData_w_Triticum__

This folder contains the files of the analysis with the discistrovirus and Picornavirus(three datasets total) and the Triticum is included in the alignment sequences

Within `Trees` folder: it includes the estimated trees of the dicistrovirus I and II viruses (the outgroup is removed)

Note: The file with label in the name represents that the label of every internal node is included in the trees

- __Dicistrovirus_I_w_triticum_wo_outgroup_aligned_w_root.tree__
- __Dicistrovirus_I_w_triticum_wo_outgroup_w_root_w_label.tree__
- __Dicistrovirus_II_w_triticum_wo_outgroup_w_root.tree__
- __Dicistrovirus_II_w_triticum_wo_outgroup_w_root_w_label.tree__
- __Picornaviridae_w_triticum_wo_outgroup_w_root.tree__
- __Picornaviridae_w_triticum_wo_outgroup_w_root_w_label.tree__


Within `Joint ancestral sequences` folder: it includes the calculated ancestral sequences at every internal nodes

(N0 represents the ancestral sequences at the rooting point)

- __Dicistrovirus_I_w_triticum_wo_outgroup_joint_ancestors.fasta__
- __Dicistrovirus_II_w_triticum_wo_outgroup_joint_ancestors.fasta__
- __Picornaviridae_w_triticum_wo_outgroup_joint_ancestors.fasta__

Within `Confidence Values of Ancestral Sequences` folder: it contains the confidence values of the estimated ancestral sequences at every internal nodes for the three datasets

- __Dicistrovirus_I_w_triticum_wo_outgroup_joint_ancestors_ConfValue.txt__
- __Dicistrovirus_II_w_triticum_wo_outgroup_joint_ancestors_ConfValue.txt__
- __Picornaviridae_w_triticum_wo_outgroup_joint_ancestors_ConfValue.txt__

Within `Ancestral sequences(N0)` folder: it includes the calculated ancestral sequences at the rooting point, which is N0.

- __Dicistrovirus_I_w_triticum_wo_outgroup_joint_ancestors.fasta__
- __Dicistrovirus_II_w_triticum_wo_outgroup_AncestralSeq.fasta__
- __Picornaviridae_w_triticum_wo_outgroup_AncestralSeq.fasta__


## __Picornviridae_N4__

This folder contains the construction of the ancestral sequence of N4 node of picornviridae, the original alignment file is "picornaviridae_w_outgroup_aligned.fasta"

There are two ways to construct N4. The first is to use all the taxons (original file), and the second way uses only two children: Encephalomyocarditis virus and Theilo virus (new file).

- __picornaviridae_aligned_N4_original.fasta__
- __picornaviridae_aligned_N4_new.fasta__


